docs/index.html

```diff
@@ -835,6 +835,18 @@ <h4 style="padding-top:40px"><a id="end-of-phrase" href="#end-of-phrase"></a>End
                     <div>
                         <pre class="notranslate"><code>- StartBeat</code></pre>
                     </div>
+                    <h4 style="padding-top:40px"><a id="p1" href="#p1"></a>P1</h4>
+                    <p><code>P1</code> marks all the notes for Player 1 in duets.
+                    </p>
+                    <div>
+                        <pre class="notranslate"><code>P1</code></pre>
+                    </div>
+                    <h4 style="padding-top:40px"><a id="p2" href="p2"></a>P2</h4>
+                    <p><code>P1</code> marks all the notes for Player 2 in duets.
+                    </p>
+                    <div>
+                        <pre class="notranslate"><code>P2</code></pre>
+                    </div>
```

Comment:
P1 & P2 have already been added in https://github.com/UltraStar-Deluxe/format/pull/62,  please remove